[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Sniffy Parent                                                      [pom]
[INFO] Sniffy Core                                                        [jar]
[INFO] Sniffy TLS Module                                                  [jar]
[INFO] Sniffy NIO Module                                                  [jar]
[INFO] Sniffy NIO Compatibility Module Dependencies                       [jar]
[INFO] Sniffy NIO Compatibility Module                                    [jar]
[INFO] Sniffy Web                                                         [jar]
[INFO] Sniffy                                                             [jar]
[INFO] sniffy-test                                                        [pom]
[INFO] sniffy-test-common                                                 [jar]
[INFO] sniffy-junit                                                       [jar]
[INFO] sniffy-kotest                                                      [jar]
[INFO] sniffy-testng                                                      [jar]
[INFO] sniffy-spring-test                                                 [jar]
[INFO] sniffy-integration-tests                                           [pom]
[INFO] sniffy-integration-tests-decrypt-tls                               [jar]
[INFO] sniffy-integration-tests-decrypt-tls-vertx                         [jar]
[INFO] sniffy-integration-tests-decrypt-tls-bc                            [jar]
[INFO] sniffy-integration-tests-decrypt-tls-bc-before-sniffy              [jar]
[INFO] sniffy-integration-tests-spock                                     [jar]
[INFO] sniffy-integration-tests-spring-boot                               [jar]
[INFO] sniffy-integration-tests-spring-boot-2                             [jar]
[INFO] sniffy-integration-tests-spring-kafka                              [jar]
[INFO] sniffy-integration-tests-tomcat                                    [jar]
[INFO] Sniffy Documentation                                               [jar]
[INFO] 
[INFO] ----------------------< io.sniffy:sniffy-parent >-----------------------
[INFO] Building Sniffy Parent 3.1.13-SNAPSHOT                            [1/25]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ sniffy-parent ---
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.7:prepare-agent (default) @ sniffy-parent ---
[INFO] argLine set to -javaagent:/home/gabsko/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/home/gabsko/breaking-updates/target/jacoco.exec,append=true
[INFO] 
[INFO] --- maven-javadoc-plugin:3.2.0:jar (attach-javadocs) @ sniffy-parent ---
[INFO] Not executing Javadoc as the project is not a Java classpath-capable package
[INFO] 
[INFO] -----------------------< io.sniffy:sniffy-core >------------------------
[INFO] Building Sniffy Core 3.1.13-SNAPSHOT                              [2/25]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ sniffy-core ---
[INFO] Deleting /home/gabsko/breaking-updates/sniffy-core/target
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ sniffy-core ---
[INFO] ShortRevision tag detected. The value is '8'.
[INFO] Executing: /bin/sh -c cd '/home/gabsko/breaking-updates/sniffy-core' && 'git' 'rev-parse' '--verify' '--short=8' 'HEAD'
[INFO] Working directory: /home/gabsko/breaking-updates/sniffy-core
[INFO] Storing buildNumber: 70e148bf at timestamp: 1675906738961
[INFO] Storing buildScmBranch: branch-sniffy/sniffy-70e148bfc4169d627ab109d7c166ad8a684cd173
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.7:prepare-agent (default) @ sniffy-core ---
[INFO] argLine set to -javaagent:/home/gabsko/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/home/gabsko/breaking-updates/sniffy-core/target/jacoco.exec,append=true
[INFO] 
[INFO] --- maven-javadoc-plugin:3.2.0:jar (attach-javadocs) @ sniffy-core ---
[ERROR] MavenReportException: Error while generating Javadoc: Unable to find javadoc command: The environment variable JAVA_HOME is not correctly set.
org.apache.maven.reporting.MavenReportException: Unable to find javadoc command: The environment variable JAVA_HOME is not correctly set.
    at org.apache.maven.plugins.javadoc.AbstractJavadocMojo.executeReport (AbstractJavadocMojo.java:2019)
    at org.apache.maven.plugins.javadoc.JavadocJar.doExecute (JavadocJar.java:199)
    at org.apache.maven.plugins.javadoc.AbstractJavadocMojo.execute (AbstractJavadocMojo.java:1935)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.io.IOException: The environment variable JAVA_HOME is not correctly set.
    at org.apache.maven.plugins.javadoc.AbstractJavadocMojo.getJavadocExecutable (AbstractJavadocMojo.java:3818)
    at org.apache.maven.plugins.javadoc.AbstractJavadocMojo.executeReport (AbstractJavadocMojo.java:2015)
    at org.apache.maven.plugins.javadoc.JavadocJar.doExecute (JavadocJar.java:199)
    at org.apache.maven.plugins.javadoc.AbstractJavadocMojo.execute (AbstractJavadocMojo.java:1935)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:294)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:960)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ sniffy-core ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ sniffy-core ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 79 source files to /home/gabsko/breaking-updates/sniffy-core/target/classes
[WARNING] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java:[3,16] sun.misc.Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java:[11,25] sun.misc.Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java:[14,9] sun.misc.Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java:[16,23] sun.misc.Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java:[18,23] sun.misc.Unsafe is internal proprietary API and may be removed in a future release
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/Sniffy.java: Some input files use or override a deprecated API.
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/Sniffy.java: Recompile with -Xlint:deprecation for details.
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java: /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java uses unchecked or unsafe operations.
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/main/java/io/sniffy/util/ReflectionUtil.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ sniffy-core ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ sniffy-core ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 42 source files to /home/gabsko/breaking-updates/sniffy-core/target/test-classes
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/test/java/io/sniffy/socket/SnifferSocketImplTest.java: Some input files use or override a deprecated API.
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/test/java/io/sniffy/socket/SnifferSocketImplTest.java: Recompile with -Xlint:deprecation for details.
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/test/java/io/sniffy/sql/PreparedStatementInvocationHandlerTest.java: Some input files use unchecked or unsafe operations.
[INFO] /home/gabsko/breaking-updates/sniffy-core/src/test/java/io/sniffy/sql/PreparedStatementInvocationHandlerTest.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ sniffy-core ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.2:test (TestNg-execution) @ sniffy-core ---
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.2:test (JUnit-execution) @ sniffy-core ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.sniffy.CoreApiExampleTest
2023-02-09 01:39:01.821+0000 [INFO] [main] [Sniffy] Initializing Sniffy 3.1.13
2023-02-09 01:39:01.848+0000 [INFO] [main] [Sniffy] sniffy.version = 3.1.13-SNAPSHOT
2023-02-09 01:39:01.848+0000 [INFO] [main] [Sniffy] sniffy.build.time = 2023-02-09 01:38
2023-02-09 01:39:01.848+0000 [INFO] [main] [Sniffy] sniffy.scm.revision = 70e148bf
2023-02-09 01:39:01.848+0000 [INFO] [main] [Sniffy] sniffy.scm.branch = branch-sniffy/sniffy-70e148bfc4169d627ab109d7c166ad8a684cd173
2023-02-09 01:39:01.849+0000 [INFO] [main] [Sniffy] OS is linux
2023-02-09 01:39:01.851+0000 [INFO] [main] [Sniffy] Java version is 11
2023-02-09 01:39:01.856+0000 [INFO] [main] [Sniffy] java.vm.vendor = Ubuntu
2023-02-09 01:39:01.856+0000 [INFO] [main] [Sniffy] java.vm.name = OpenJDK 64-Bit Server VM
2023-02-09 01:39:01.856+0000 [INFO] [main] [Sniffy] java.vm.version = 11.0.17+8-post-Ubuntu-1ubuntu218.04
2023-02-09 01:39:01.856+0000 [INFO] [main] [Sniffy] java.version = 11.0.17
2023-02-09 01:39:01.856+0000 [INFO] [main] [Sniffy] java.version.date = 2022-10-18
2023-02-09 01:39:01.856+0000 [INFO] [main] [Sniffy] os.name = Linux
2023-02-09 01:39:01.857+0000 [INFO] [main] [Sniffy] os.arch = amd64
2023-02-09 01:39:01.857+0000 [INFO] [main] [Sniffy] os.version = 5.4.0-113-generic
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.669 s - in io.sniffy.CoreApiExampleTest
[INFO] Running io.sniffy.MockDriverTest
[INFO] Tests run: 16, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.103 s - in io.sniffy.MockDriverTest
[INFO] Running io.sniffy.SnifferExpectApiTest
[ERROR] Tests run: 7, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.031 s <<< FAILURE! - in io.sniffy.SnifferExpectApiTest
[ERROR] testExactlyOneOtherThreadInsert(io.sniffy.SnifferExpectApiTest)  Time elapsed: 0.009 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testExactlyOneOtherThreadInsert(SnifferExpectApiTest.java:32)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testExactlyOneOtherThreadInsert(SnifferExpectApiTest.java:32)

[ERROR] testExactlyTwoInsertsAnyThread(io.sniffy.SnifferExpectApiTest)  Time elapsed: 0.003 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testExactlyTwoInsertsAnyThread(SnifferExpectApiTest.java:41)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testExactlyTwoInsertsAnyThread(SnifferExpectApiTest.java:41)

[ERROR] testNotMoreThanOneInsert(io.sniffy.SnifferExpectApiTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testNotMoreThanOneInsert(SnifferExpectApiTest.java:17)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testNotMoreThanOneInsert(SnifferExpectApiTest.java:17)

[ERROR] testExactlyOneThreadLocalInsert(io.sniffy.SnifferExpectApiTest)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testExactlyOneThreadLocalInsert(SnifferExpectApiTest.java:25)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferExpectApiTest.testExactlyOneThreadLocalInsert(SnifferExpectApiTest.java:25)

[INFO] Running io.sniffy.SnifferListenersTest
[ERROR] Tests run: 5, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.039 s <<< FAILURE! - in io.sniffy.SnifferListenersTest
[ERROR] testQueryFromOtherThread(io.sniffy.SnifferListenersTest)  Time elapsed: 0.003 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferListenersTest.testQueryFromOtherThread(SnifferListenersTest.java:116)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferListenersTest.testQueryFromOtherThread(SnifferListenersTest.java:116)

[INFO] Running io.sniffy.SnifferParseQueryTest
Exception in thread "Thread-4" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Merge.prepare(Merge.java:279)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:68)
	... 2 more
Exception in thread "Thread-11" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-15" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Merge.prepare(Merge.java:279)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:68)
	... 2 more
[ERROR] Tests run: 20, Failures: 1, Errors: 5, Skipped: 0, Time elapsed: 0.102 s <<< FAILURE! - in io.sniffy.SnifferParseQueryTest
[ERROR] testAtMostMergeOtherThreadNegative(io.sniffy.SnifferParseQueryTest)  Time elapsed: 0.005 s  <<< FAILURE!
java.lang.AssertionError: Expected exception: io.sniffy.WrongNumberOfQueriesError

[ERROR] testNeverInsertNegative(io.sniffy.SnifferParseQueryTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.WrongNumberOfQueriesError> but was<java.lang.RuntimeException>
	at io.sniffy.SnifferParseQueryTest.testNeverInsertNegative(SnifferParseQueryTest.java:19)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testNeverInsertNegative(SnifferParseQueryTest.java:19)

[ERROR] testNeverInsertOtherThreadPositive(io.sniffy.SnifferParseQueryTest)  Time elapsed: 0.004 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testNeverInsertOtherThreadPositive(SnifferParseQueryTest.java:27)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testNeverInsertOtherThreadPositive(SnifferParseQueryTest.java:27)

[ERROR] testAtMostMergePositive(io.sniffy.SnifferParseQueryTest)  Time elapsed: 0.003 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testAtMostMergePositive(SnifferParseQueryTest.java:75)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testAtMostMergePositive(SnifferParseQueryTest.java:75)

[ERROR] testAtMostMergeNegative(io.sniffy.SnifferParseQueryTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.WrongNumberOfQueriesError> but was<java.lang.RuntimeException>
	at io.sniffy.SnifferParseQueryTest.testAtMostMergeNegative(SnifferParseQueryTest.java:86)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testAtMostMergeNegative(SnifferParseQueryTest.java:86)

[ERROR] testAtMostMergeOtherThreadPositive(io.sniffy.SnifferParseQueryTest)  Time elapsed: 0.011 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testAtMostMergeOtherThreadPositive(SnifferParseQueryTest.java:95)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SnifferParseQueryTest.testAtMostMergeOtherThreadPositive(SnifferParseQueryTest.java:95)

[INFO] Running io.sniffy.SnifferTest
[ERROR] Tests run: 30, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.131 s <<< FAILURE! - in io.sniffy.SnifferTest
[ERROR] testVerifyExactUpdate(io.sniffy.SnifferTest)  Time elapsed: 0.005 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferTest.testVerifyExactUpdate(SnifferTest.java:94)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SnifferTest.testVerifyExactUpdate(SnifferTest.java:94)

[INFO] Running io.sniffy.SniffyStackoverflowApplicationTests
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.072 s - in io.sniffy.SniffyStackoverflowApplicationTests
[INFO] Running io.sniffy.SniffyTest
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.521 s - in io.sniffy.SniffyTest
[INFO] Running io.sniffy.SpyNetworkTrafficTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.003 s - in io.sniffy.SpyNetworkTrafficTest
[INFO] Running io.sniffy.SpyParseQueryTest
Exception in thread "Thread-31" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Merge.prepare(Merge.java:279)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:68)
	... 2 more
Exception in thread "Thread-38" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-42" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Merge.prepare(Merge.java:279)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:68)
	... 2 more
[ERROR] Tests run: 20, Failures: 1, Errors: 5, Skipped: 0, Time elapsed: 0.043 s <<< FAILURE! - in io.sniffy.SpyParseQueryTest
[ERROR] testAtMostMergeOtherThreadNegative(io.sniffy.SpyParseQueryTest)  Time elapsed: 0.005 s  <<< FAILURE!
java.lang.AssertionError: Expected exception: io.sniffy.WrongNumberOfQueriesError

[ERROR] testNeverInsertNegative(io.sniffy.SpyParseQueryTest)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.WrongNumberOfQueriesError> but was<java.lang.RuntimeException>
	at io.sniffy.SpyParseQueryTest.testNeverInsertNegative(SpyParseQueryTest.java:17)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SpyParseQueryTest.testNeverInsertNegative(SpyParseQueryTest.java:17)

[ERROR] testNeverInsertOtherThreadPositive(io.sniffy.SpyParseQueryTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SpyParseQueryTest.testNeverInsertOtherThreadPositive(SpyParseQueryTest.java:25)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.SpyParseQueryTest.testNeverInsertOtherThreadPositive(SpyParseQueryTest.java:25)

[ERROR] testAtMostMergePositive(io.sniffy.SpyParseQueryTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SpyParseQueryTest.testAtMostMergePositive(SpyParseQueryTest.java:73)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SpyParseQueryTest.testAtMostMergePositive(SpyParseQueryTest.java:73)

[ERROR] testAtMostMergeNegative(io.sniffy.SpyParseQueryTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.WrongNumberOfQueriesError> but was<java.lang.RuntimeException>
	at io.sniffy.SpyParseQueryTest.testAtMostMergeNegative(SpyParseQueryTest.java:84)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SpyParseQueryTest.testAtMostMergeNegative(SpyParseQueryTest.java:84)

[ERROR] testAtMostMergeOtherThreadPositive(io.sniffy.SpyParseQueryTest)  Time elapsed: 0.004 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SpyParseQueryTest.testAtMostMergeOtherThreadPositive(SpyParseQueryTest.java:93)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.SpyParseQueryTest.testAtMostMergeOtherThreadPositive(SpyParseQueryTest.java:93)

[INFO] Running io.sniffy.SpyWithValueTest
[INFO] Tests run: 18, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.077 s - in io.sniffy.SpyWithValueTest
[INFO] Running io.sniffy.ThreadLocalSnifferTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.006 s - in io.sniffy.ThreadLocalSnifferTest
[INFO] Running io.sniffy.configuration.SniffyConfigurationTest
[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.002 s - in io.sniffy.configuration.SniffyConfigurationTest
[INFO] Running io.sniffy.registry.ConnectionsRegistryTest
2023-02-09 01:39:03.604+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with null
2023-02-09 01:39:03.604+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was null
2023-02-09 01:39:03.605+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@114b81af
2023-02-09 01:39:03.611+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with null
2023-02-09 01:39:03.611+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was null
2023-02-09 01:39:03.612+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@50f88cea
2023-02-09 01:39:03.631+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with null
2023-02-09 01:39:03.631+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was null
2023-02-09 01:39:03.631+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@1d2cffea
java.net.SocketException: Connection reset
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:186)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:200)
	at io.sniffy.socket.EchoServerRule$SocketInputStreamReader.run(EchoServerRule.java:193)
	at java.base/java.lang.Thread.run(Thread.java:829)
[INFO] Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.051 s - in io.sniffy.registry.ConnectionsRegistryTest
[INFO] Running io.sniffy.socket.CaptureDataInputStreamTrafficTest
2023-02-09 01:39:04.649+0000 [INFO] [main] [Sniffy] Socket monitoring enabled - installing SnifferSocketImplFactory
2023-02-09 01:39:04.650+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was io.sniffy.socket.SnifferSocketImplFactory@1d2cffea
2023-02-09 01:39:04.650+0000 [INFO] [main] [Sniffy] NIO monitoring enabled - loading NIO Sniffy Module
2023-02-09 01:39:04.651+0000 [ERROR] [main] [Sniffy] java.lang.ClassNotFoundException: io.sniffy.nio.SniffySelectorProviderModule
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:315)
	at io.sniffy.Sniffy.loadNioModule(Sniffy.java:294)
	at io.sniffy.Sniffy.access$100(Sniffy.java:47)
	at io.sniffy.Sniffy$3.propertyChange(Sniffy.java:232)
	at java.desktop/java.beans.PropertyChangeSupport.fire(PropertyChangeSupport.java:341)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:334)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:266)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:308)
	at io.sniffy.configuration.SniffyConfiguration.setMonitorNio(SniffyConfiguration.java:218)
	at io.sniffy.socket.CaptureDataInputStreamTrafficTest.testCaptureInputStream(CaptureDataInputStreamTrafficTest.java:80)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.038 s - in io.sniffy.socket.CaptureDataInputStreamTrafficTest
[INFO] Running io.sniffy.socket.CaptureTrafficOverviewTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.006 s - in io.sniffy.socket.CaptureTrafficOverviewTest
[INFO] Running io.sniffy.socket.CaptureTrafficTest
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.036 s - in io.sniffy.socket.CaptureTrafficTest
[INFO] Running io.sniffy.socket.SnifferInputStreamTest
[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.602 s - in io.sniffy.socket.SnifferInputStreamTest
[INFO] Running io.sniffy.socket.SnifferOutputStreamTest
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.014 s - in io.sniffy.socket.SnifferOutputStreamTest
[INFO] Running io.sniffy.socket.SnifferSocketImplFactoryTest
2023-02-09 01:39:05.356+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with null
2023-02-09 01:39:05.357+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was null
2023-02-09 01:39:05.357+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@26c9521a
2023-02-09 01:39:05.358+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with null
2023-02-09 01:39:05.360+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with null
2023-02-09 01:39:05.360+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was io.sniffy.socket.SnifferSocketImplFactoryTest$TestSocketImplFactory@2ef08ad0
2023-02-09 01:39:05.361+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@2fab6393
2023-02-09 01:39:05.364+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with io.sniffy.socket.SnifferSocketImplFactoryTest$TestSocketImplFactory@2ef08ad0
2023-02-09 01:39:05.365+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was io.sniffy.socket.SnifferSocketImplFactoryTest$TestSocketImplFactory@2ef08ad0
2023-02-09 01:39:05.366+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@572b031f
2023-02-09 01:39:05.367+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with io.sniffy.socket.SnifferSocketImplFactoryTest$TestSocketImplFactory@2ef08ad0
2023-02-09 01:39:05.369+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was io.sniffy.socket.SnifferSocketImplFactoryTest$TestSocketImplFactory@2ef08ad0
2023-02-09 01:39:05.369+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@60d5c16
2023-02-09 01:39:05.370+0000 [INFO] [main] [SnifferSocketImplFactory] Uninstalling SniffySocketImplFactory - replacing with io.sniffy.socket.SnifferSocketImplFactoryTest$TestSocketImplFactory@2ef08ad0
2023-02-09 01:39:05.371+0000 [INFO] [main] [SnifferSocketImplFactory] Original SocketImplFactory was io.sniffy.socket.SnifferSocketImplFactoryTest$TestSocketImplFactory@2ef08ad0
2023-02-09 01:39:05.371+0000 [INFO] [main] [SnifferSocketImplFactory] SocketImplFactory set to io.sniffy.socket.SnifferSocketImplFactory@21d297a
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.021 s - in io.sniffy.socket.SnifferSocketImplFactoryTest
[INFO] Running io.sniffy.socket.SnifferSocketImplReflectionTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in io.sniffy.socket.SnifferSocketImplReflectionTest
[INFO] Running io.sniffy.socket.SnifferSocketImplTest
[INFO] Tests run: 32, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.099 s - in io.sniffy.socket.SnifferSocketImplTest
[INFO] Running io.sniffy.socket.TcpExpectationTest
[INFO] Tests run: 18, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.159 s - in io.sniffy.socket.TcpExpectationTest
[INFO] Running io.sniffy.sql.DataSourceRegistryTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in io.sniffy.sql.DataSourceRegistryTest
[INFO] Running io.sniffy.sql.PreparedStatementInvocationHandlerTest
[ERROR] Tests run: 7, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.083 s <<< FAILURE! - in io.sniffy.sql.PreparedStatementInvocationHandlerTest
[ERROR] testExecuteBatchCountUpdatedRows(io.sniffy.sql.PreparedStatementInvocationHandlerTest)  Time elapsed: 0.002 s  <<< ERROR!
org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, ?) [42122-206]
	at io.sniffy.sql.PreparedStatementInvocationHandlerTest.testExecuteBatchCountUpdatedRows(PreparedStatementInvocationHandlerTest.java:98)

[ERROR] testExecuteBatch(io.sniffy.sql.PreparedStatementInvocationHandlerTest)  Time elapsed: 0 s  <<< ERROR!
org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, ?) [42122-206]
	at io.sniffy.sql.PreparedStatementInvocationHandlerTest.testExecuteBatch(PreparedStatementInvocationHandlerTest.java:86)

[ERROR] testExecuteEmptyBatch(io.sniffy.sql.PreparedStatementInvocationHandlerTest)  Time elapsed: 0.001 s  <<< ERROR!
org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, ?) [42122-206]
	at io.sniffy.sql.PreparedStatementInvocationHandlerTest.testExecuteEmptyBatch(PreparedStatementInvocationHandlerTest.java:113)

[ERROR] testExecuteInsertPreparedStatement(io.sniffy.sql.PreparedStatementInvocationHandlerTest)  Time elapsed: 0 s  <<< ERROR!
org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, ?) [42122-206]
	at io.sniffy.sql.PreparedStatementInvocationHandlerTest.testExecuteInsertPreparedStatement(PreparedStatementInvocationHandlerTest.java:122)

[INFO] Running io.sniffy.sql.QueryTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in io.sniffy.sql.QueryTest
[INFO] Running io.sniffy.sql.SniffyDataSourceTest
[ERROR] Tests run: 10, Failures: 0, Errors: 7, Skipped: 0, Time elapsed: 0.028 s <<< FAILURE! - in io.sniffy.sql.SniffyDataSourceTest
[ERROR] testGetConnectionWithSocketOperation(io.sniffy.sql.SniffyDataSourceTest)  Time elapsed: 0.004 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Cannot mock/spy class org.h2.jdbcx.JdbcDataSource
Mockito cannot mock/spy because :
 - final class
	at io.sniffy.sql.SniffyDataSourceTest.testGetConnectionWithSocketOperation(SniffyDataSourceTest.java:101)

[ERROR] testGetPooledConnectionWithCredentialsWithSocketOperation(io.sniffy.sql.SniffyDataSourceTest)  Time elapsed: 0 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Cannot mock/spy class org.h2.jdbcx.JdbcDataSource
Mockito cannot mock/spy because :
 - final class
	at io.sniffy.sql.SniffyDataSourceTest.testGetPooledConnectionWithCredentialsWithSocketOperation(SniffyDataSourceTest.java:269)

[ERROR] testGetXAConnectionWithSocketOperation(io.sniffy.sql.SniffyDataSourceTest)  Time elapsed: 0.001 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Cannot mock/spy class org.h2.jdbcx.JdbcDataSource
Mockito cannot mock/spy because :
 - final class
	at io.sniffy.sql.SniffyDataSourceTest.testGetXAConnectionWithSocketOperation(SniffyDataSourceTest.java:153)

[ERROR] testGetConnectionWithCredentialsWithSocketOperation(io.sniffy.sql.SniffyDataSourceTest)  Time elapsed: 0 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Cannot mock/spy class org.h2.jdbcx.JdbcDataSource
Mockito cannot mock/spy because :
 - final class
	at io.sniffy.sql.SniffyDataSourceTest.testGetConnectionWithCredentialsWithSocketOperation(SniffyDataSourceTest.java:127)

[ERROR] testGetPooledConnectionWithSocketOperation(io.sniffy.sql.SniffyDataSourceTest)  Time elapsed: 0 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Cannot mock/spy class org.h2.jdbcx.JdbcDataSource
Mockito cannot mock/spy because :
 - final class
	at io.sniffy.sql.SniffyDataSourceTest.testGetPooledConnectionWithSocketOperation(SniffyDataSourceTest.java:205)

[ERROR] testGetXAConnectionWithCredentialsWithSocketOperation(io.sniffy.sql.SniffyDataSourceTest)  Time elapsed: 0 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Cannot mock/spy class org.h2.jdbcx.JdbcDataSource
Mockito cannot mock/spy because :
 - final class
	at io.sniffy.sql.SniffyDataSourceTest.testGetXAConnectionWithCredentialsWithSocketOperation(SniffyDataSourceTest.java:179)

[ERROR] testGetPooledConnectionClosedIfRejected(io.sniffy.sql.SniffyDataSourceTest)  Time elapsed: 0 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Cannot mock/spy class org.h2.jdbcx.JdbcDataSource
Mockito cannot mock/spy because :
 - final class
	at io.sniffy.sql.SniffyDataSourceTest.testGetPooledConnectionClosedIfRejected(SniffyDataSourceTest.java:231)

[INFO] Running io.sniffy.sql.SniffyDriverTest
[INFO] Tests run: 22, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.075 s - in io.sniffy.sql.SniffyDriverTest
[INFO] Running io.sniffy.sql.SqlQueriesTest
[ERROR] Tests run: 21, Failures: 0, Errors: 5, Skipped: 0, Time elapsed: 0.061 s <<< FAILURE! - in io.sniffy.sql.SqlQueriesTest
[ERROR] testExactTwoAnyThreadsInsertQueries(io.sniffy.sql.SqlQueriesTest)  Time elapsed: 0.003 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoAnyThreadsInsertQueries(SqlQueriesTest.java:153)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoAnyThreadsInsertQueries(SqlQueriesTest.java:153)

[ERROR] testExactTwoInsertQueries(io.sniffy.sql.SqlQueriesTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoInsertQueries(SqlQueriesTest.java:48)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoInsertQueries(SqlQueriesTest.java:48)

[ERROR] testExactTwoAnyThreadsMergeQueries(io.sniffy.sql.SqlQueriesTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoAnyThreadsMergeQueries(SqlQueriesTest.java:177)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoAnyThreadsMergeQueries(SqlQueriesTest.java:177)

[ERROR] testExactTwoMergeQueries(io.sniffy.sql.SqlQueriesTest)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoMergeQueries(SqlQueriesTest.java:75)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoMergeQueries(SqlQueriesTest.java:75)

[ERROR] testExactTwoSelectQueries(io.sniffy.sql.SqlQueriesTest)  Time elapsed: 0.003 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoSelectQueries(SqlQueriesTest.java:41)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueriesTest.testExactTwoSelectQueries(SqlQueriesTest.java:41)

[INFO] Running io.sniffy.sql.SqlQueries_Rows_Test
Exception in thread "Thread-240" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-244" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-247" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-249" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-251" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-254" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-255" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Merge.prepare(Merge.java:279)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:68)
	... 2 more
Exception in thread "Thread-257" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-260" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-263" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
Exception in thread "Thread-268" java.lang.RuntimeException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:83)
	at io.sniffy.BaseTest.lambda$executeStatementsInOtherThread$0(BaseTest.java:103)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:248)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:230)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.prepare(Insert.java:300)
	at org.h2.command.Parser.prepareCommand(Parser.java:971)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:624)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:562)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:143)
	at jdk.internal.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.sniffy.sql.JdbcInvocationHandler.invokeTargetImpl(JdbcInvocationHandler.java:93)
	at io.sniffy.sql.StatementInvocationHandler.invokeTargetAndRecord(StatementInvocationHandler.java:109)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:76)
	at io.sniffy.sql.StatementInvocationHandler.invokeImpl(StatementInvocationHandler.java:20)
	at io.sniffy.sql.JdbcInvocationHandler.invoke(JdbcInvocationHandler.java:56)
	at com.sun.proxy.$Proxy19.executeUpdate(Unknown Source)
	at io.sniffy.BaseTest.executeStatements(BaseTest.java:59)
	... 2 more
[ERROR] Tests run: 64, Failures: 6, Errors: 31, Skipped: 0, Time elapsed: 0.124 s <<< FAILURE! - in io.sniffy.sql.SqlQueries_Rows_Test
[ERROR] testOneRowNoneQueries_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfQueriesError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowNoneQueries_Exception(SqlQueries_Rows_Test.java:275)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowNoneQueries_Exception(SqlQueries_Rows_Test.java:275)

[ERROR] testOneRowAnyThreads_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfRowsError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowAnyThreads_Exception(SqlQueries_Rows_Test.java:267)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowAnyThreads_Exception(SqlQueries_Rows_Test.java:267)

[ERROR] testExactQueriesAnyThreadsDeleteMaxMinRows(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.007 s  <<< FAILURE!
io.sniffy.sql.WrongNumberOfRowsError: 
Expected between 2 and 4deleted rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

	at io.sniffy.sql.SqlQueries_Rows_Test.testExactQueriesAnyThreadsDeleteMaxMinRows(SqlQueries_Rows_Test.java:204)

[ERROR] testExactQueriesAtMostOneRow(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testExactQueriesAtMostOneRow(SqlQueries_Rows_Test.java:248)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testExactQueriesAtMostOneRow(SqlQueries_Rows_Test.java:248)

[ERROR] testTwoRowsTwoQueriesUpdateAnyThreads(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsTwoQueriesUpdateAnyThreads(SqlQueries_Rows_Test.java:324)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsTwoQueriesUpdateAnyThreads(SqlQueries_Rows_Test.java:324)

[ERROR] testOneDeleteQueryCurrentThreadBetweenTwoAndThreeRows(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenTwoAndThreeRows(SqlQueries_Rows_Test.java:442)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenTwoAndThreeRows(SqlQueries_Rows_Test.java:442)

[ERROR] testExactThreeRowsDeleteOtherThreadsMinMaxQueries(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.006 s  <<< FAILURE!
io.sniffy.sql.WrongNumberOfRowsError: 
Expected between 3 and 3 other threadsdeleted rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

	at io.sniffy.sql.SqlQueries_Rows_Test.testExactThreeRowsDeleteOtherThreadsMinMaxQueries(SqlQueries_Rows_Test.java:184)

[ERROR] testBetweenNineAndElevenRowSelectedAllQueries_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfRowsError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testBetweenNineAndElevenRowSelectedAllQueries_Exception(SqlQueries_Rows_Test.java:170)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testBetweenNineAndElevenRowSelectedAllQueries_Exception(SqlQueries_Rows_Test.java:170)

[ERROR] testOneMergeRow(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneMergeRow(SqlQueries_Rows_Test.java:36)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneMergeRow(SqlQueries_Rows_Test.java:36)

[ERROR] testNoneRowsOtherThreadMinMaxQueriesOther(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testNoneRowsOtherThreadMinMaxQueriesOther(SqlQueries_Rows_Test.java:531)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testNoneRowsOtherThreadMinMaxQueriesOther(SqlQueries_Rows_Test.java:531)

[ERROR] testOneDeleteQueryCurrentThreadBetweenMaxMinRows_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfRowsError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenMaxMinRows_Exception(SqlQueries_Rows_Test.java:460)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenMaxMinRows_Exception(SqlQueries_Rows_Test.java:460)

[ERROR] testOneQueryOtherThreadsDeleteExactTwoRows(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneQueryOtherThreadsDeleteExactTwoRows(SqlQueries_Rows_Test.java:573)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneQueryOtherThreadsDeleteExactTwoRows(SqlQueries_Rows_Test.java:573)

[ERROR] testExactQueriesExactRows(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< FAILURE!
io.sniffy.sql.WrongNumberOfRowsError: 
Expected between 3 and 3 current threadreturned / affected rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

	at io.sniffy.sql.SqlQueries_Rows_Test.testExactQueriesExactRows(SqlQueries_Rows_Test.java:223)

[ERROR] testOneQueryOneRowCurrentThreadUpdate(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneQueryOneRowCurrentThreadUpdate(SqlQueries_Rows_Test.java:547)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneQueryOneRowCurrentThreadUpdate(SqlQueries_Rows_Test.java:547)

[ERROR] testAtMostOneDeleteQueryReturnsValidMaxMinRows(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneDeleteQueryReturnsValidMaxMinRows(SqlQueries_Rows_Test.java:376)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneDeleteQueryReturnsValidMaxMinRows(SqlQueries_Rows_Test.java:376)

[ERROR] testTwoRowsAnyThreadUpdateMinMaxQueries(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsAnyThreadUpdateMinMaxQueries(SqlQueries_Rows_Test.java:483)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsAnyThreadUpdateMinMaxQueries(SqlQueries_Rows_Test.java:483)

[ERROR] testOneRowMerge(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowMerge(SqlQueries_Rows_Test.java:308)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowMerge(SqlQueries_Rows_Test.java:308)

[ERROR] testMaxRowsInserted(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testMaxRowsInserted(SqlQueries_Rows_Test.java:145)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testMaxRowsInserted(SqlQueries_Rows_Test.java:145)

[ERROR] testAtMostOneUpdateQueryReturnsValidMinMaxRows(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneUpdateQueryReturnsValidMinMaxRows(SqlQueries_Rows_Test.java:367)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneUpdateQueryReturnsValidMinMaxRows(SqlQueries_Rows_Test.java:367)

[ERROR] testAtMostOneRowUpdated_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfRowsError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneRowUpdated_Exception(SqlQueries_Rows_Test.java:152)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneRowUpdated_Exception(SqlQueries_Rows_Test.java:152)

[ERROR] testTwoRowsMinMaxQueries(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsMinMaxQueries(SqlQueries_Rows_Test.java:291)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsMinMaxQueries(SqlQueries_Rows_Test.java:291)

[ERROR] testExactQueriesBetweenRowsAnyThreads(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.005 s  <<< FAILURE!
io.sniffy.sql.WrongNumberOfQueriesError: 
Expected between 5 and 5 queries
Observed 3 queries instead:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo');
DELETE FROM PUBLIC.PROJECT;

	at io.sniffy.sql.SqlQueries_Rows_Test.testExactQueriesBetweenRowsAnyThreads(SqlQueries_Rows_Test.java:232)

[ERROR] testTwoQueryMaxTwoMergeRowsOtherThreads(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.003 s  <<< FAILURE!
io.sniffy.sql.WrongNumberOfQueriesError: 
Expected between 2 and 2 other threads queries
Observed 1 queries instead:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar');

	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoQueryMaxTwoMergeRowsOtherThreads(SqlQueries_Rows_Test.java:65)

[ERROR] testTwoRowsMaxMinQueries_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.003 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfQueriesError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsMaxMinQueries_Exception(SqlQueries_Rows_Test.java:299)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsMaxMinQueries_Exception(SqlQueries_Rows_Test.java:299)

[ERROR] testAtMostOneSelectQueryReturnsAtMostOneRow(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneSelectQueryReturnsAtMostOneRow(SqlQueries_Rows_Test.java:358)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testAtMostOneSelectQueryReturnsAtMostOneRow(SqlQueries_Rows_Test.java:358)

[ERROR] testOneDeleteQueryCurrentThreadBetweenMinMaxRows(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenMinMaxRows(SqlQueries_Rows_Test.java:451)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenMinMaxRows(SqlQueries_Rows_Test.java:451)

[ERROR] testResultSetMethods(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testResultSetMethods(SqlQueries_Rows_Test.java:79)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testResultSetMethods(SqlQueries_Rows_Test.java:79)

[ERROR] testOneRowOneQuerySelectOtherThread(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOneQuerySelectOtherThread(SqlQueries_Rows_Test.java:315)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOneQuerySelectOtherThread(SqlQueries_Rows_Test.java:315)

[ERROR] testThreeRowsDeleteQueriesMinMax(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testThreeRowsDeleteQueriesMinMax(SqlQueries_Rows_Test.java:407)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testThreeRowsDeleteQueriesMinMax(SqlQueries_Rows_Test.java:407)

[ERROR] testExactThreeRowsDeleteOtherThreads(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.004 s  <<< FAILURE!
io.sniffy.sql.WrongNumberOfRowsError: 
Expected between 3 and 3 other threadsdeleted rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

	at io.sniffy.sql.SqlQueries_Rows_Test.testExactThreeRowsDeleteOtherThreads(SqlQueries_Rows_Test.java:164)

[ERROR] testOneQueryAnyThreadsDeleteAtMostOneRow(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneQueryAnyThreadsDeleteAtMostOneRow(SqlQueries_Rows_Test.java:564)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneQueryAnyThreadsDeleteAtMostOneRow(SqlQueries_Rows_Test.java:564)

[ERROR] testThreeRowsDeleteQueriesMaxMin_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfQueriesError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testThreeRowsDeleteQueriesMaxMin_Exception(SqlQueries_Rows_Test.java:417)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testThreeRowsDeleteQueriesMaxMin_Exception(SqlQueries_Rows_Test.java:417)

[ERROR] testTwoRowsTwoQueriesMergeAnyThreads(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsTwoQueriesMergeAnyThreads(SqlQueries_Rows_Test.java:334)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testTwoRowsTwoQueriesMergeAnyThreads(SqlQueries_Rows_Test.java:334)

[ERROR] testOneRowOtherThreadMaxMinQueriesDelete(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOtherThreadMaxMinQueriesDelete(SqlQueries_Rows_Test.java:513)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOtherThreadMaxMinQueriesDelete(SqlQueries_Rows_Test.java:513)

[ERROR] testOneRowOtherThreadMinMaxQueriesDelete(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOtherThreadMinMaxQueriesDelete(SqlQueries_Rows_Test.java:522)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOtherThreadMinMaxQueriesDelete(SqlQueries_Rows_Test.java:522)

[ERROR] testOneRowOneQueryInsertOtherThreads(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.RuntimeException: 
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOneQueryInsertOtherThreads(SqlQueries_Rows_Test.java:283)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneRowOneQueryInsertOtherThreads(SqlQueries_Rows_Test.java:283)

[ERROR] testOneOtherRow_Exception(io.sniffy.sql.SqlQueries_Rows_Test)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.Exception: Unexpected exception, expected<io.sniffy.sql.WrongNumberOfRowsError> but was<java.lang.RuntimeException>
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneOtherRow_Exception(SqlQueries_Rows_Test.java:45)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: 
Column "SEQ_PROJECT.NEXTVAL" not found; SQL statement:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo') [42122-206]
	at io.sniffy.sql.SqlQueries_Rows_Test.testOneOtherRow_Exception(SqlQueries_Rows_Test.java:45)

[INFO] Running io.sniffy.sql.SqlUtilTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in io.sniffy.sql.SqlUtilTest
[INFO] Running io.sniffy.util.ExceptionUtilTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in io.sniffy.util.ExceptionUtilTest
[INFO] Running io.sniffy.util.IOUtilTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in io.sniffy.util.IOUtilTest
[INFO] Running io.sniffy.util.ReflectionFieldCopierTest
java.lang.NoSuchFieldException: baz
	at java.base/java.lang.Class.getDeclaredField(Class.java:2411)
	at io.sniffy.util.ReflectionFieldCopier.getFieldImpl(ReflectionFieldCopier.java:30)
	at io.sniffy.util.ReflectionFieldCopier.<init>(ReflectionFieldCopier.java:10)
	at io.sniffy.util.ReflectionFieldCopierTest.notAvailableField(ReflectionFieldCopierTest.java:39)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in io.sniffy.util.ReflectionFieldCopierTest
[INFO] Running io.sniffy.util.StackTraceExtractorTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in io.sniffy.util.StackTraceExtractorTest
[INFO] Running io.sniffy.util.StringUtilTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in io.sniffy.util.StringUtilTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   SnifferParseQueryTest.testAtMostMergeOtherThreadNegative Expected exception: io.sniffy.WrongNumberOfQueriesError
[ERROR]   SpyParseQueryTest.testAtMostMergeOtherThreadNegative Expected exception: io.sniffy.WrongNumberOfQueriesError
[ERROR]   SqlQueries_Rows_Test.testExactQueriesAnyThreadsDeleteMaxMinRows:204 Expected between 2 and 4deleted rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

[ERROR]   SqlQueries_Rows_Test.testExactQueriesBetweenRowsAnyThreads:232 Expected between 5 and 5 queries
Observed 3 queries instead:
INSERT INTO PUBLIC.PROJECT (ID, NAME) VALUES (SEQ_PROJECT.NEXTVAL, 'foo');
DELETE FROM PUBLIC.PROJECT;

[ERROR]   SqlQueries_Rows_Test.testExactQueriesExactRows:223 Expected between 3 and 3 current threadreturned / affected rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

[ERROR]   SqlQueries_Rows_Test.testExactThreeRowsDeleteOtherThreads:164 Expected between 3 and 3 other threadsdeleted rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

[ERROR]   SqlQueries_Rows_Test.testExactThreeRowsDeleteOtherThreadsMinMaxQueries:184 Expected between 3 and 3 other threadsdeleted rows 
Observed 0 rows instead:
DELETE FROM PUBLIC.PROJECT; /* 0 rows */

[ERROR]   SqlQueries_Rows_Test.testTwoQueryMaxTwoMergeRowsOtherThreads:65 Expected between 2 and 2 other threads queries
Observed 1 queries instead:
MERGE INTO PUBLIC.PROJECT (ID, NAME) KEY (ID) VALUES (SEQ_PROJECT.NEXTVAL, 'bar');

[ERROR] Errors: 
[ERROR]   SnifferExpectApiTest.testExactlyOneOtherThreadInsert:32->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferExpectApiTest.testExactlyOneThreadLocalInsert:25->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferExpectApiTest.testExactlyTwoInsertsAnyThread:41->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferExpectApiTest.testNotMoreThanOneInsert:17->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferListenersTest.testQueryFromOtherThread:116->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferParseQueryTest.testAtMostMergeNegative »  Unexpected exception, expecte...
[ERROR]   SnifferParseQueryTest.testAtMostMergeOtherThreadPositive:95->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferParseQueryTest.testAtMostMergePositive:75->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferParseQueryTest.testNeverInsertNegative »  Unexpected exception, expecte...
[ERROR]   SnifferParseQueryTest.testNeverInsertOtherThreadPositive:27->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SnifferTest.testVerifyExactUpdate:94->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SpyParseQueryTest.testAtMostMergeNegative »  Unexpected exception, expected<io...
[ERROR]   SpyParseQueryTest.testAtMostMergeOtherThreadPositive:93->BaseTest.executeStatements:83 » Runtime
[ERROR]   SpyParseQueryTest.testAtMostMergePositive:73->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SpyParseQueryTest.testNeverInsertNegative »  Unexpected exception, expected<io...
[ERROR]   SpyParseQueryTest.testNeverInsertOtherThreadPositive:25->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   PreparedStatementInvocationHandlerTest.testExecuteBatch:86 » JdbcSQLSyntaxError
[ERROR]   PreparedStatementInvocationHandlerTest.testExecuteBatchCountUpdatedRows:98 » JdbcSQLSyntaxError
[ERROR]   PreparedStatementInvocationHandlerTest.testExecuteEmptyBatch:113 » JdbcSQLSyntaxError
[ERROR]   PreparedStatementInvocationHandlerTest.testExecuteInsertPreparedStatement:122 » JdbcSQLSyntaxError
[ERROR]   SniffyDataSourceTest.testGetConnectionWithCredentialsWithSocketOperation:127 Mockito
[ERROR]   SniffyDataSourceTest.testGetConnectionWithSocketOperation:101 Mockito 
Cannot ...
[ERROR]   SniffyDataSourceTest.testGetPooledConnectionClosedIfRejected:231 Mockito 
Cann...
[ERROR]   SniffyDataSourceTest.testGetPooledConnectionWithCredentialsWithSocketOperation:269 Mockito
[ERROR]   SniffyDataSourceTest.testGetPooledConnectionWithSocketOperation:205 Mockito 
C...
[ERROR]   SniffyDataSourceTest.testGetXAConnectionWithCredentialsWithSocketOperation:179 Mockito
[ERROR]   SniffyDataSourceTest.testGetXAConnectionWithSocketOperation:153 Mockito 
Canno...
[ERROR]   SqlQueriesTest.testExactTwoAnyThreadsInsertQueries:153->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueriesTest.testExactTwoAnyThreadsMergeQueries:177->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueriesTest.testExactTwoInsertQueries:48->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueriesTest.testExactTwoMergeQueries:75->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueriesTest.testExactTwoSelectQueries:41->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testAtMostOneDeleteQueryReturnsValidMaxMinRows:376->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testAtMostOneRowUpdated_Exception »  Unexpected exception...
[ERROR]   SqlQueries_Rows_Test.testAtMostOneSelectQueryReturnsAtMostOneRow:358->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testAtMostOneUpdateQueryReturnsValidMinMaxRows:367->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testBetweenNineAndElevenRowSelectedAllQueries_Exception » 
[ERROR]   SqlQueries_Rows_Test.testExactQueriesAtMostOneRow:248->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testMaxRowsInserted:145->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testNoneRowsOtherThreadMinMaxQueriesOther:531->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenMaxMinRows_Exception » 
[ERROR]   SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenMinMaxRows:451->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneDeleteQueryCurrentThreadBetweenTwoAndThreeRows:442->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneMergeRow:36->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneOtherRow_Exception »  Unexpected exception, expect...
[ERROR]   SqlQueries_Rows_Test.testOneQueryAnyThreadsDeleteAtMostOneRow:564->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneQueryOneRowCurrentThreadUpdate:547->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneQueryOtherThreadsDeleteExactTwoRows:573->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneRowAnyThreads_Exception »  Unexpected exception, e...
[ERROR]   SqlQueries_Rows_Test.testOneRowMerge:308->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneRowNoneQueries_Exception »  Unexpected exception, ...
[ERROR]   SqlQueries_Rows_Test.testOneRowOneQueryInsertOtherThreads:283->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneRowOneQuerySelectOtherThread:315->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneRowOtherThreadMaxMinQueriesDelete:513->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testOneRowOtherThreadMinMaxQueriesDelete:522->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testResultSetMethods:79->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testThreeRowsDeleteQueriesMaxMin_Exception »  Unexpected ...
[ERROR]   SqlQueries_Rows_Test.testThreeRowsDeleteQueriesMinMax:407->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testTwoRowsAnyThreadUpdateMinMaxQueries:483->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testTwoRowsMaxMinQueries_Exception »  Unexpected exceptio...
[ERROR]   SqlQueries_Rows_Test.testTwoRowsMinMaxQueries:291->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testTwoRowsTwoQueriesMergeAnyThreads:334->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[ERROR]   SqlQueries_Rows_Test.testTwoRowsTwoQueriesUpdateAnyThreads:324->BaseTest.executeStatement:45->BaseTest.executeStatements:83 » Runtime
[INFO] 
[ERROR] Tests run: 366, Failures: 8, Errors: 63, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Sniffy Parent 3.1.13-SNAPSHOT:
[INFO] 
[INFO] Sniffy Parent ...................................... SUCCESS [  0.393 s]
[INFO] Sniffy Core ........................................ FAILURE [ 11.733 s]
[INFO] Sniffy TLS Module .................................. SKIPPED
[INFO] Sniffy NIO Module .................................. SKIPPED
[INFO] Sniffy NIO Compatibility Module Dependencies ....... SKIPPED
[INFO] Sniffy NIO Compatibility Module .................... SKIPPED
[INFO] Sniffy Web ......................................... SKIPPED
[INFO] Sniffy ............................................. SKIPPED
[INFO] sniffy-test ........................................ SKIPPED
[INFO] sniffy-test-common ................................. SKIPPED
[INFO] sniffy-junit ....................................... SKIPPED
[INFO] sniffy-kotest ...................................... SKIPPED
[INFO] sniffy-testng ...................................... SKIPPED
[INFO] sniffy-spring-test ................................. SKIPPED
[INFO] sniffy-integration-tests ........................... SKIPPED
[INFO] sniffy-integration-tests-decrypt-tls ............... SKIPPED
[INFO] sniffy-integration-tests-decrypt-tls-vertx ......... SKIPPED
[INFO] sniffy-integration-tests-decrypt-tls-bc ............ SKIPPED
[INFO] sniffy-integration-tests-decrypt-tls-bc-before-sniffy SKIPPED
[INFO] sniffy-integration-tests-spock ..................... SKIPPED
[INFO] sniffy-integration-tests-spring-boot ............... SKIPPED
[INFO] sniffy-integration-tests-spring-boot-2 ............. SKIPPED
[INFO] sniffy-integration-tests-spring-kafka .............. SKIPPED
[INFO] sniffy-integration-tests-tomcat .................... SKIPPED
[INFO] Sniffy Documentation ............................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.731 s
[INFO] Finished at: 2023-02-09T01:39:10Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (JUnit-execution) on project sniffy-core: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/gabsko/breaking-updates/sniffy-core/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :sniffy-core
